modules/service-networking/README.md

```diff
@@ -0,0 +1,28 @@
+# Terraform Google service networking
+
+This module creates global network address and a service networking
```

Comment:
Could we add some more context on the uses cases? IIUC this module would be used for private service access. Is there any other usecase cc @imrannayer

Reply:
Updated the module README and example README